docs/serving/feature-flags.md

```diff
@@ -275,6 +275,30 @@ spec:
         ...
 ```
 
+## Kubernetes Security Context Capabilities
+
+* **Type**: extension
+* **ConfigMap key**: `kubernetes.podspec-addcapabilities`
+
+This flag controls whether users can add capabilities on the Pod's securityContext.
+
+When set to "enabled" or "allowed" it allows [Linux capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html) to be added to the container.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
+spec:
+ template:
+  spec:
+   containers:
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+            ...
+```
```

Comment:
Can we make this a full sample just to avoid confusion? It seems truncated?

Reply:
A full sample is okay with me. Though it looks like most of the other yaml examples in this doc are also snippets rather than full samples -- would it be better to make a separate PR to update all of them?